static/app/utils/profiling/profile/importProfile.tsx

```diff
@@ -127,6 +142,60 @@ function importChromeTrace(
   throw new Error('Failed to parse trace input format');
 }
 
+function importSentrySampledProfile(
+  input: Profiling.SentrySampledProfile,
+  options: ImportOptions
+): ProfileGroup {
+  const frameIndex = createSentrySampleProfileFrameIndex(input.profile.frames);
+  const samplesByThread: Record<
+    string,
+    Profiling.SentrySampledProfile['profile']['samples']
+  > = {};
+
+  for (let i = 0; i < input.profile.samples.length; i++) {
+    const sample = input.profile.samples[i];
+    if (!samplesByThread[sample.thread_id]) {
+      samplesByThread[sample.thread_id] = [];
+    }
+    samplesByThread[sample.thread_id].push(sample);
+  }
+
+  for (const key in samplesByThread) {
+    samplesByThread[key].sort(
+      (a, b) =>
+        parseInt(a.elapsed_since_start_ns, 10) - parseInt(b.elapsed_since_start_ns, 10)
+    );
+  }
+
+  const profiles: Profile[] = [];
+
+  for (const key in samplesByThread) {
+    profiles.push(
+      wrapWithSpan(
+        options.transaction,
+        () =>
+          SentrySampledProfile.FromProfile(
+            samplesByThread[key],
+            input.profile.stacks,
+            frameIndex
+          ),
+        {
+          op: 'profile.import',
+          description: 'evented',
+        }
+      )
+    );
+  }
+
+  return {
+    traceID: '',
+    name: '',
+    activeProfileIndex: 0,
+    metadata: {},
```

Comment:
we need to copy over the metadata fields from our profile, else parts of the UI will display unknown data even though we have it on the profile. For example, this relies on all of the metadata mostly
<img width="292" alt="CleanShot 2022-09-28 at 14 44 51@2x" src="https://user-images.githubusercontent.com/9317857/192863674-5f5fd91a-f99e-44ba-b782-405c36128fd0.png">

Reply:
how do we want to handle multiple transactions? right now i'm just taking the first one... 
![image](https://user-images.githubusercontent.com/7349258/192879520-0ffe3adb-f75f-4645-b7c1-ed249f57bcb8.png)


we also don't have things like `organizationID` and `projectID` but i don't really think its necessary, that will be known upfront. the following fields are what i'm unable to fill;
![image](https://user-images.githubusercontent.com/7349258/192879753-8d0e7ec9-5123-436a-85a7-6660881a406e.png)

Reply:
There will always be 1 transaction when we return the profile. The array was to handle multiple transactions at ingestion. Selecting the first one is fine.

`classification` should be a field from `device` now. It also won't be present when we're looking at a backend profile so you'll need to handle that case. If it's not there already, it's a bug and I'll fix that.

`android_api_level` won't be in the sample format as Android doesn't use it.

For `organization_id`/`project_id`, I think we can remove them from this metadata? They're not really profile related.

For `received`, you should use `timestamp`. Instead of being "when the backend received the profile", `timestamp` is "when the device sent the profile", matching the field in a transaction. We still have `received` but we're not sending it back anymore. I think it's good to match whatever is done in the transaction details so that's why we started reporting `timestamp` only.